packages/calcite-components/src/components/flow-item/flow-item.stories.ts

```diff
@@ -279,13 +279,13 @@ export const withActionBarAndContentTop_TestOnly = (): string =>
     </calcite-flow-item>
   </div>`;
 
-export const footerPaddingAndContentBottom_TestOnly = (): string =>
+export const footerStartEndAndContentBottom_TestOnly = (): string =>
```

Comment:
Can you merge the content from both `footerStartEndAndContentBottom_TestOnly` and `footerPaddingAndContentBottom_TestOnly` stories? The latest adds coverage for the start/end footer slots, but loses the footer one. 

I think we also need one that demonstrates the footer slot priority (`footer` > `footer-start`/`footer-end`).

☝️ applies to panel stories too.